tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -329,6 +332,12 @@ private async void HandleAdaptiveAction(RenderedAdaptiveCard sender, AdaptiveAct
         // https://github.com/microsoft/devhome/issues/644
     }
 
+    public void AnnounceDeleted()

```

Comment:
I know it's gross to  Application.Current.GetService the ScreenReaderService in WidgetControl, but can we do this there?

Reply:
Done